Makefile

```diff
@@ -135,6 +143,30 @@ local-cross: bin/skopeo.darwin.amd64 bin/skopeo.linux.arm bin/skopeo.linux.arm64
 build-container:
 	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -t "$(IMAGE)" .
 
+# Build container specific to arch
+build-arch-container:
+	${CONTAINER_RUNTIME} build -t "${ARCH_IMAGE}" contrib/skopeoimage/${SOURCE_TYPE}
+
+# Push container specific to arch
+push-arch-container:
+	echo "${QUAY_PASSWORD}" | ${CONTAINER_RUNTIME} login ${REGISTRY} -u "${QUAY_USERNAME}" --password-stdin
+	${CONTAINER_RUNTIME} push "${ARCH_IMAGE}"
+ifneq ($(EXTRA_REPO),)
+	${CONTAINER_RUNTIME} tag "${ARCH_IMAGE}" "${EXTRA_ARCH_IMAGE}"
+	${CONTAINER_RUNTIME} push "${EXTRA_ARCH_IMAGE}"
+endif
+
+# Push manifest to make multi-arch image
+push-ma-manifest:
+	echo "${QUAY_PASSWORD}" | docker login ${REGISTRY} -u "${QUAY_USERNAME}" --password-stdin
+	sudo chmod 0755 /etc/docker
+	DOCKER_CLI_EXPERIMENTAL="enabled" docker manifest create "${MA_IMAGE}" $(foreach arch,${ARCHES}, ${MA_IMAGE}-${arch})
```

Comment:
Podman experts, should we be using Docker for all of this?

Reply:
docker is default container tool for travis. podman and buildah, as far as I know, are questionable to install with some hacks (especially for non amd64 archs), see https://travis-ci.community/t/podman-libpod-support/6823/6. I can play with it, if it's hard requirement, however would prefer to use docker, as it's more stable and verified approach for Travis

Reply:
I'd prefer podman, but @barthy1 is right, travis at this point defaults pretty hard to docker from what I've seen.  @edsantiago might have other thoughts.